language_detector/main.py

```diff
@@ -4,5 +4,27 @@
 
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    words = text.split()
+    
+    language_guess = ''
+    valid_word_max = 0
+    num_words = {}
```

Comment:
I don't understand why this is here. I don't see any other reference to this object num_words. 

Is it necessary?